src/components/StoreInfo/StoreInfoContainer.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { useState } from 'react';
+import classnames from 'classnames';
+import styled from 'styled-components';
+import { Location, BrowsePageSeller } from '../../utilities';
+import StoreStory from '../StoreStory';
+import StoreMenu from './StoreMenuTab';
+import StoreGallery from './StoreGalleryTab';
+import StoreMerch from './StoreMerchTab';
+import styles from './styles.module.scss';
+import defaultStoreFront from './misc-store.png';
+
+import { getStoreImages } from './StoreImages';
+
+type Props = {
+  seller: BrowsePageSeller;
+};
+
+export const StoreInfo: React.SFC<Props> = ({ seller }) => {
+  const { summary, story, cuisine_name, locations } = seller;
+
+  const storeNavItems = ['story', 'menu', 'gallery', 'share'];
+
+  const [currentMerchantView, setMerchantView] = useState('story');
+
+  const switchMerchantView = () => {
+    switch (currentMerchantView) {
+      case 'gallery':
+        return <StoreGallery seller={seller} />;
+      case 'menu':
+        return <StoreMenu seller={seller} getStoreImages={getStoreImages} />;
+      case 'merch':
+        return <StoreMerch seller={seller} getStoreImages={getStoreImages} />;
+      case 'share':
+        // TODO: plug in the share page that Dan creates
```

Comment:
FYI @danthaman44

Reply:
Great! Since the new share icons haven't been moved over yet, we can hide the "share" tab for now